packages/trader/src/Modules/Contract/Containers/contract-replay.jsx

```diff
@@ -278,9 +277,17 @@ const ReplayChart = connect(({ modules, ui, common }) => {
     const contract_store = contract_replay.contract_store;
     const contract_config = contract_store.contract_config;
     const is_chart_ready = contract_replay.is_chart_ready;
+    /**
```

Comment:
Can you add a `TODO` here to remove once **DBot** supports dark theme? 🙏🏻

Reply:
@zaki-binary just added it.